config/mutant.yml

```diff
@@ -1,3 +1,6 @@
 ---
 name: promise
 namespace: Promise
+ignore_subjects:
+- "Promise::Observer#promise_rejected"
+- "Promise::Observer#promise_fulfilled"
```

Comment:
I didn't even know about the `ignore_subjects` mutant config.  However, if the purpose for these tests are to have default methods so that observer can just implement one of these, then they could be tested using custom observers that don't override both of these methods, thus actually test the default no-op behaviour.  If you would prefer to require a Promise::Observer to implement both of these methods, then the methods could just be removed and turned into comments, e.g.

```diff
diff --git a/lib/promise/observer.rb b/lib/promise/observer.rb
index 904daca..d6ffd00 100644
--- a/lib/promise/observer.rb
+++ b/lib/promise/observer.rb
@@ -1,11 +1,12 @@
 class Promise
   # The `Promise::Observer` module allows an object to be
-  # notified of `Promise` state changes.
+  # notified of `Promise` state changes by implementing
+  # the following methods
+  #
+  # * def promise_fulfilled(value, on_fulfill_arg)
+  # * def promise_rejected(reason, on_reject_arg)
   #
   # See `Promise#observe`.
   module Observer
-    def promise_fulfilled(_value, _on_fulfill_arg); end
-
-    def promise_rejected(_reason, _on_reject_arg); end
   end
 end
```

Reply:
@dylanahsmith Good point. I'll add tests for observers that only implement one or the other method